### PR TITLE
Updated `cc.sh` to work on Linux and added a readme entry for a test dependency

### PR DESCRIPTION
**Pull request by EECOLOR**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.kateproject*
```
Comment (EECOLOR) [inline .gitignore]: I'll remove this one... It was still in that old branch

File: bin/cc.sh [hunk 2]
```diff
@@ -7,6 +7,6 @@ goroot="$(dirname "$bindir")"
 what=${1:-test}
 shift
 
-which -s spy || go get github.com/jpillora/spy
```
Comment (paulp) [inline bin/cc.sh:10]: The -s suppresses the output, which I require. If -s isn't on linux then how about `which spy >/dev/null`.

Comment (EECOLOR) [inline bin/cc.sh:10]: That would work, I'll fix it (probably) tomorrow

**Comment by EECOLOR** [comment]
I added your suggestion

**Comment by paulp** [comment]
I squashed it and merged that.

**Pull request closed without merging**